### ISSUE TITLE
Can not get all image urls in post which has over 5 images

### ISSUE THREAD

**Issue by thocao276** [kevinzg/facebook-scraper#286]
In a post that has over 5 images, facebook just shows 5 first images and the other will be disabled with the plus sign in the last image. The get_posts function can not get all images from this post. It can get images that are shown. How can I get all images in posts?

**Comment by neon-ninja** [comment]
It would get all images in high quality in this case. Check the `images` key

**Comment by thocao276** [comment]
Thank you for your response,
I realize I can see posts in the group I am crawling without login. But `images` need logging in.
After logged in, I just get only the first 5 images not all of the images in the post.
There isn't any option or solution to crawl all of the images in the post.
BG.

**Comment by neon-ninja** [comment]
What group / posts are you having this problem with?

**Comment by thocao276** [comment]
This post `https://www.facebook.com/groups/1853667948249427/permalink/2972476306368580/`

**Comment by neon-ninja** [comment]
With the latest master branch, the code:

```python
print(len(next(get_posts(
    post_urls=["https://m.facebook.com/groups/1853667948249427/permalink/2972476306368580/"],
    cookies="cookies.txt"
))["images"]))
```
returns 6, with or without cookies. Might be time for me to do another release

**Comment by neon-ninja** [comment]
released as v0.2.38
